.swiftlint.yml

```diff
@@ -0,0 +1,4 @@
+disabled_rules: # rule identifiers to exclude from running
+  - trailing_whitespace
```

Comment:
I personally like trailing whitespace between functions which this rule gives a warning for